.changeset/silly-jeans-collect.md

```diff
@@ -24,3 +24,5 @@ Add `captureWhitespaceSensitivity` configuration option.
   {% endfor %}
 {% endcapture %}
 ```
+
+Thank you @stinjs96 for this feature!
```

Comment:
@charlespwd Oops... It should be @stijns96

Reply:
Fixed in 07166da